api-server/common/models/user.js

```diff
@@ -838,7 +840,8 @@ export default function(User) {
           points: progressTimestamps.length,
           completedChallenges,
           ...getProgress(progressTimestamps, timezone),
-          ...normaliseUserFields(user)
+          ...normaliseUserFields(user),
+          joinDate: user._id.getTimeStamp()
```

Comment:
```suggestion
          joinDate: user.id.getTimeStamp()
```
I'm not sure why it's `id` and not `_id`, but it is.

Reply:
I get `user.id.getTimeStamp is not a function` for id
@Manish-Giri, it would be great if you could look into this.

Reply:
@ahmadabdolsaheb I was able to finally sign in with a local user using Auth0, looking into this now.

Reply:
@ojeytonwilliams Adding `joinDate` to `getSessionUser()` worked, many thanks!